search.nim

```diff
@@ -305,7 +305,7 @@ proc findText*(win: var utils.MainWin, forward: bool) =
     highlightAll(win, pattern, true, win.autoSettings.search)
   else:
     # Stop it from highlighting due to selection.
-    win.tabs[currentTab].highlighted = newHighlightAll("", true, -1)
+    win.tabs[currentTab].highlighted = newHighlightAll("", true, guint.high)
```

Comment:
Why is this change required?

Reply:
If something down the line needs a guint you should change it there, not up here.

Reply:
Well, the newHighlightAll procedure simply sets the field idleID to the applied value. idleID is used in one procedure call, and obtained from another, both being glib procedures which expects a guint. So there's really no reason why this should be represented as an int in the Nim code just to be converted every time it's actually used. The reason why guint.high is required is simply that Nim is a bit more restrictive than C which is fine with assigning -1 to a uint. In Nim you can't do this without explicitly casting, but the value of -1 cast to a guint and guint.high should be the same.

Reply:
Meh, alright.